mammoth/distributed/components.py

```diff
@@ -87,7 +87,50 @@ def needs_communication(self) -> bool:
         return self.group is not None
 
 
-# TODO: This is a misnomer: Not an entire XCoder, but just one AttentionLayers block
+@dataclass  # type: ignore
+class DistributedTransformerWrapper(DistributedComponent, ABC):
```

Comment:
docstring. It's fairly self-evident but making it clear what is the purpose of this cls is is bound to help the next person to work on is